data-serving/data-service/test/controllers/case.test.ts

```diff
@@ -492,6 +493,47 @@ describe('POST', () => {
             unchangedCaseUploadIds[1],
         );
     });
+    it('geocodes everything that is necessary', async () => {
+        await request(app)
+            .post('/api/geocode/seed')
+            .send({
+                country: 'Canada',
+                geoResolution: 'Country',
+                geometry: { latitude: 42.42, longitude: 11.11 },
+                name: 'Canada',
+            })
+            .expect(200);
+        await request(app)
+            .post('/api/geocode/seed')
+            .send({
+                administrativeAreaLevel1: 'Quebec',
+                country: 'Canada',
+                geoResolution: 'Admin1',
+                geometry: { latitude: 33.33, longitude: 99.99 },
+                name: 'Montreal',
+            })
+            .expect(200);
+        await request(app)
+            .post('/api/cases')
+            .send({
+                ...caseMustGeocode,
+                ...curatorMetadata,
+            })
+            .expect(201)
+            .expect('Content-Type', /json/);
+        expect(await Case.collection.countDocuments()).toEqual(1);
```

Comment:
Can you check that the created case has the right geocoded fields?

Reply:
yup, done.